Plugins/Public/base_plugin/PlayerCommands.cpp

```diff
@@ -1882,6 +1882,243 @@ namespace PlayerCommands
 		}
 	}
 
+	bool CheckSolarDistances(uint client, uint systemID, Vector pos)
+	{
+		// Other POB Check
+		for (const auto& base : player_bases)
+		{
+			if (base.second->basetype != "legacy" 
+				|| base.second->invulnerable 
+				|| !base.second->logic
+				|| base.second->system != systemID
+				|| (base.second->position.x == pos.x

```

Comment:
Why do we want to allow two bases at exactly the same spot?

Reply:
It's a bit of a hacky way to detect the base being compared with itself without passing an extra parameter to the method, and skipping processing if that's the case since distance would be 0

Reply:
lol ok